content/en/docs/tasks/manage-kubernetes-objects/storage-version-migration.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Migrate Kubernetes Objects Using Storage Version Migration
+
+reviewers:
+  - deads2k
+  - jpbetz
+  - enj
+  - nilekhc
+
+content_type: task
+min-kubernetes-server-version: v1.30
+weight: 60
+---
+
+<!-- overview -->
+{{< feature-state feature_gate_name="StorageVersionMigration" >}}
+
+Kubernetes relies on API data being actively re-written, to support some 
+maintenance activities related to at rest storage. Two prominent examples are 
+the versioned schema of stored resources (that is, the preferred storage schema 
+changing from v1 to v2 for a given resource) and encryption at rest
+(that is, rewriting stale data based on a change in how the data should be encrypted).
+
+## {{% heading "prerequisites" %}}
+
+Install [`kubectl`](/docs/tasks/tools/#kubectl).
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+<!-- steps -->
+
+## Re-encrypt Kubernetes secrets using storage version migration
```

Comment:
What's the link between storage version migration and re-encryption? Are we sure that readers can get that link?